### DISCUSSION TITLE
使用paddledet训练的模型在ppstructure中调用出错

### DISCUSSION THREAD

**Discussion by Irving11330** [PaddlePaddle/PaddleOCR#12411] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Win10专业版 python3.7.9
- 版本号/Version：Paddle：2.5.0  PaddleOCR：2.6.1 问题相关组件/Related components：ppstructure;paddledetection
- 运行指令/Command Code：使用python脚本调用ppstructure识别图纸
- 完整报错/Complete Error Message：ValueError: (InvalidArgument) The input of Op(Conv) should be a 4-D or 5-D Tensor. But received: input's dimension is 1, input's shape is [0].
  [Hint: Expected in_dims.size() == 4 || in_dims.size() == 5 == true, but received in_dims.size() == 4 || in_dims.size() == 5:0 != true:1.] (at ..\paddle\fluid\operators\fused\conv_fusion_op.cc:236)
  [operator < conv2d_fusion > error]

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

使用paddledet中picodet_lcnet_x1_0_layout模型迁徙训练picodet_lcnet_x1_0_fgd_layout_cdla.pdparams，使用单卡训练，学习率减少至默认值的1/8
训练模型配置文件修改如下：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/63030153/897f12f1-ac7a-4705-a821-3e8b627d8a97)
fgd蒸馏配置文件修改如下：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/63030153/3889b3c5-8f26-4a51-89ce-dedb177a1472)
使用python脚本调用ppstructure识别图纸，注释掉paddleocr.py第630行的maybe_download(params.layout_model_dir, layout_url)，再将在paddledet中导出的模型复制到原默认模型位置，结果报错：
ValueError: (InvalidArgument) The input of Op(Conv) should be a 4-D or 5-D Tensor. But received: input's dimension is 1, input's shape is [0].
  [Hint: Expected in_dims.size() == 4 || in_dims.size() == 5 == true, but received in_dims.size() == 4 || in_dims.size() == 5:0 != true:1.] (at ..\paddle\fluid\operators\fused\conv_fusion_op.cc:236)
  [operator < conv2d_fusion > error]
值得一提的是原本picodet_lcnet_x1_0_fgd_layout_cdla_infer模型中的.pdmodel文件2699kb，经过迁徙训练导出的model.pdmodel只有244kb

**Comment by adzhua** [comment]
请问这个怎么解决的

**Reply by Andy197527** [reply]
解决了吗

**Reply by NBclasss** [reply]
请问解决了吗。求教
